src/components/Question.jsx

```diff
@@ -0,0 +1,39 @@
+import classNames from "classnames";
+import Answer from "./Answer";
+
+export default function Question({ question, check, answerQuestion }) {
+  return (
+    <div>
+      <h1 className="font-bold text-[#293264]">{question.question}</h1>
+      <div className="flex gap-2 flex-wrap mt-1">
+        {question.randomAnswers.map((el, i) => {
+          const picked = question.chosenAnswer === el;
+          const correct = el === question["correct_answer"];
+          const classes = classNames(
+            {
+              "bg-[#94D7A2]": check && correct,
+            },
+            { "bg-[#F8BCBC]": check && (picked && !correct) },
+            {
+              "border-[#4d5b9e7e] text-[#29326477]":
+                check && !picked && !correct,
+            },
+            { "border-[#4D5B9E] ": !check && !picked },
+            { "border-[#4D5B9E] bg-[#D6DBF5]": !check && picked }
```

Comment:
You don't need to wrap every class utility inside an object, you can use single object and write all of them in.
```javascript

classname({
    "bg-[#94D7A2]": check && correct,
    "bg-[#F8BCBC]": check && (picked && !correct),
    ....
})
```